content/en/graphing/widgets/slo.md

```diff
@@ -0,0 +1,79 @@
+---
+title: SLO Widget
+kind: documentation
+description: "Track your SLOs."
+further_reading:
+- link: "https://www.datadoghq.com/blog/slo-monitoring-widget/"
+  tag: "Blog"
+  text: "Track the status of your SLOs with the new monitor uptime widget"
+aliases:
+ - /monitors/monitor_uptime_widget/
+ - /monitors/slo_widget/
+---
+
+<div class="alert alert-info">
+Service Level Objectives are now generally available.
+</div>
+
+{{< img src="graphing/widgets/slo/summary_editor.png" alt="monitor uptime widget" responsive="true" >}}
+
+Use the SLO and uptime widget to track your SLOs (Service Level Objectives) and uptime on screenboards and timeboards. Go to Datadog’s [Service Level Objectives page][1] to create new SLOs and view all existing ones. Select an existing SLO from the dropdown and display it on any dashboard.
+
+Any time window that’s been configured for your SLOs will be available for viewing on the widget, but you can select and deselect them depending on what you want to see. You can also enable calendar time windows like a week-to-date, month-to-date, previous week, and the previous month. These will be made available depending on the time windows you have previously configured. For instance, the 7-day window will enable week-to-date while the 30-day one will enable month-to-date, etc. Additional calendar windows will be available as the underlying data permits.
+
+To reconfigure your targets or enable more time windows, edit the source SLO. 
+
+SLOs in the preview can be clicked to open their related detail panels.
+
+*Uptime* defined as the amount of time a monitor was in an *up* state (OK) compared to *down* state (non-OK). Similar to the monitor status page visualization, the status is represented in bars as green (up) and red (down). Example: ’99 % of the time latency is less than 200ms.`
+
+You can also track success rate and event-based SLIs (Service Level Indicators). Example: ’99 % of requests are successful.`
+
+### Viewing options
+
+For monitor based SLOs, you can select between displaying the monitor’s uptime percentage, its worst-performing groups, or both. Those options are also available for SLOs aggregating multiple monitors so you can view those monitors individually instead of the groups. 
```

Comment:
```suggestion
For monitor based SLOs, you can select between displaying the monitor’s uptime percentage, its worst-performing groups, or both. These options are also available for SLOs aggregating multiple monitors so you can view the monitors individually instead of the groups. 
```